test/rubygems/test_gem_text.rb

```diff
@@ -71,4 +71,9 @@ def test_levenshtein_distance_replace
     assert_equal 7, levenshtein_distance("xxxxxxx", "ZenTest")
     assert_equal 7, levenshtein_distance("zentest", "xxxxxxx")
   end
+
+  def test_levenshtein_distance_long
+    assert_equal 13, levenshtein_distance("cat", "thundercatsarego")
+    assert_equal 13, levenshtein_distance("thundercatsarego", "cat")
```

Comment:
I guess it's better to put these assertions into `test_levenshtein_distance_remove`.